rust/extractor/src/qltest.rs

```diff
@@ -54,6 +54,7 @@ path = "main.rs"
 fn set_sources(config: &mut Config) -> anyhow::Result<()> {
     let path_iterator = glob("**/*.rs").context("globbing test sources")?;
     config.inputs = path_iterator
+        .filter(|f| f.is_err() || !f.as_ref().unwrap().starts_with("target"))
```

Comment:
I guess your intention is to keep `Err` paths, right? So any errors are not silently ignored. If you just want to drop `Err` paths then it might be nicer to use `f.is_ok_and()` instead of the `is_err` check and `unwrap`ping.

Reply:
yeah, I hesitated between the two, but in the end opted for not suppressing the errors for now